caldp/tests/test_all.py

```diff
@@ -3,12 +3,14 @@
 """
 import os
 import subprocess
+import tempfile
```

Comment:
Good style (Python pep8) breaks imports up into 3 sections in this order:  Python packages,  3rd party packages,  our packages.    There should be an empty line between sections.  You're also supposed to have only one package/module per line,  not comma separated lists which are syntactically correct but frowned on.   This is formatting Black cannot fix because it doesn't know the module origins.   I'm no saint myself,  I sometimes use lists of packages...  but generally/should avoid them.

Reply:
Thanks, I did not know about this convention actually. I am bad about just importing anything anywhere.